python/dune/xt/common/timings.py

```diff
@@ -6,7 +6,12 @@
 #      or  GPL-2.0+ (http://opensource.org/licenses/gpl-license)
```

Comment:
Out of curiosity, did you explicitly `git mv` this and edit the file afterwards, or did git just decide this was a rename?

Reply:
I think I moved it with `git mv` several times (back and forth), and edited it at the same time... :/